src/components/Extra.vue

```diff
@@ -978,8 +956,6 @@
     <!-- --------------------------------------------------------------------------------------------- -->
 </template>
 
-<script>
-export default {
-    name: 'Extra',
-}
+<script lang="ts" setup>
+import ElementsPanelVue from './Panels/ElementsPanel/ElementsPanel.vue'
```

Comment:
lets not use the 'vue' suffix in component names?

Reply:
removed it.